src/Permission.cxx

```diff
@@ -89,7 +89,7 @@ initPermissions(const ConfigData &config)
 		permission_default = 0;
 
 		param.With([](const char *value){
-			const char *separator = strchr(value,
+			const char *separator = std::strchr(value,
```

Comment:
Missing include

Reply:
It's included in string_view, coming from util/StringView.hxx

Speaking of which, that file is funny. Why is there a C++ version check? MPD does not compile without C++17.

Reply:
Does the standard guarantee that `<string_view>` implies `<cstring>` or is that an implementation detail?

But we shouldn't rely on indirect includes by our own headers; these are certainly implementation details.

`StringView.hxx` has a version check because I shared this files with various projects, some of which must be buildable with GCC 6, but for "pure" C++17 projects like MPD, it is completely obsolete and is fading out.